kie-drools-wb/kie-drools-wb-webapp/pom.xml

```diff
@@ -523,6 +523,13 @@
       <artifactId>kie-wb-common-workbench-backend</artifactId>
     </dependency>
 
+    <!-- Drools Workbench Services -->
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-wb-verifier-api</artifactId>
+      <scope>provided</scope>
```

Comment:
Should this really be `provided`? It seems that the other `-api` modules re not being marked as such. (I may be overlooking something, but it just caught my eye...)